keyvault-argo.tf

```diff
@@ -0,0 +1,108 @@
+# This was throwing a duplicate resource name error, so I appended -extend to the alias
+data "azurerm_client_config" "current-extended" {}
```

Comment:
Yeah, because it's already defined in keyvault.tf file. You can delete this line.

Reply:
Just deleted it! 👍